privacyidea/static/components/token/views/token.enroll.html

```diff
@@ -28,6 +28,7 @@ <h3 translate>Enroll a new token</h3>
         <div class="form-group" ng-hide="loggedInUser.role == 'user' && !checkRight('setdescription')">
             <label for="description" translate>Description</label>
             <input type="text" class="form-control" id="description"
+                   autocomplete="new-password"
```

Comment:
I think it was not necessary here, since it is a description field?

Reply:
I double-checked. Apparently, it is needed, but I also cannot see why.